packages/DataGrid/src/DataGrid.js

```diff
@@ -105,19 +105,18 @@ const DataGrid = React.forwardRef((props, ref) => {
   const refTotalCanGrow = React.useRef(0);
 
   const [scrollBarWidth, setScrollBarWidth] = React.useState(0);
-  const [gridShouldHaveFocus, setGridShouldHaveFocus] = React.useState(true);
   const [pageSize, setPageSize] = React.useState(null);
   const i18n = useI18n();
   // these two value are sensitive in Grids with lots of columns and can degradate performance alot.
   // be caution when using them.
-  const overscanRowCount = 5;
-  const overscanColumnCount = 2;
+  const overscanRowCount = 7;
+  const overscanColumnCount = 5;
 
   const rowCount = data.length;
   const {
     "DataGrid.ColumnDefinition": extractedColumnDefinitions,
     "DataGrid.Basement": Basement,
-    "DataGrid.InfiniteScroll": InfiniteScroll,
+    "DataGrid.InfiniteScroll": extractedInfiniteScroll,
```

Comment:
rename infiniteScro for extractedInfintiveScroll to no be confused with the import of the `Infinitive` component.